README.md

```diff
@@ -94,6 +97,183 @@ see [Developer’s guide to auth with Azure Resource Manager API](http://aka.ms/
 After creating the service principal, you should have three pieces of information, a client id (GUID), client secret
 (string) and tenant id (GUID) or domain name (string).
 
+# Azure Multiple API versions & Profiles
+
+With 0.15.0 of Azure SDK, multiple API versions and profiles are introduced. With these changes, each individual gem
+has multiple versions of the services and several profiles. The azure sdk rollup gem also consists of several profiles. The following section provides details on the usage of multiple API versions and profiles.
```

Comment:
Should we write `azure_sdk` as it's the name of the gem?

Reply:
Done